rust/sdk-processor/src/utils/starting_version.rs

```diff
@@ -49,12 +49,17 @@ pub async fn get_starting_version(
 pub async fn get_min_last_success_version_parquet(
     indexer_processor_config: &IndexerProcessorConfig,
     conn_pool: ArcDbPool,
-    processor_names: Vec<String>,
+    table_names: Vec<String>,
 ) -> Result<u64> {
-    let min_processed_version =
-        get_min_processed_version_from_db(conn_pool.clone(), processor_names)
+    let min_processed_version = if indexer_processor_config.backfill_config.is_some() {
```

Comment:
for backfill, we can reuse the existing behavior to get the min_processsed_version since its using backfill alias